frontend/src/routing/routesNames.ts

```diff
@@ -0,0 +1,15 @@
+export enum RoutesNames {
```

Comment:
```suggestion
export enum Route {
```
enum is a type, not collection, soo good practice is to name it with singular form